File: src/main/java/loxal/epvin/datastore/model/Resource.java
/*
 * Copyright 2012 Alexander Orlov <alexander.orlov@loxal.net>. All rights reserved.
 */

package loxal.epvin.datastore.model;

import com.googlecode.objectify.annotation.Entity;

/**
 * @author Alexander Orlov <alexander.orlov@loxal.net>
 */
@Entity
public class Resource extends DatastoreEntity {
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    // employee / tennis court
    private String type;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: src/main/kotlin/loxal/epvin/datastore/model/Resource.kt
/*
 * Copyright 2012 Alexander Orlov <alexander.orlov@loxal.net>. All rights reserved.
 */

package loxal.epvin.datastore.model

import com.googlecode.objectify.annotation.Entity

@Entity
public class Resource : DatastoreEntity() {
    var name: String = ""

    // employee / tennis court
    var type: String = ""
}
